torch/testing/_internal/common_jit.py

```diff
@@ -72,10 +72,10 @@ def clone_inputs(requires_grad):
 
     with enable_profiling_mode_for_profiling_tests():
         # test single grad case
-        outputs = self.runAndSaveRNG(reference_func, recording_inputs, kwargs)
+        outputs = output_func(self.runAndSaveRNG(reference_func, recording_inputs, kwargs))
```

Comment:
Now I'm thinking it might be good to put `output_func` into `self.runAndSaveRNG`? Then we don't have to wrap every instance of `runAndSaveRNG` with `output_func`.

Reply:
I agree all of the wrapping isn't pretty- I could make a wrapper around runAndSaveRNG that can take in the output_func as a parameter? I'm hesitant to change runAndSaveRNG itself because it's used in several other places

Reply:
We can probably leave it as is then.

Reply:
When we add an "input_func" we should consider renaming these functions to "gradcheck_input_func" and "gradcheck_output_func" for clarity.